target/linux/ath79/dts/ar9344_qxwlan_e750a-v4-16m.dts

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344_qxwlan_e750x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
```

Comment:
These two includes can be dropped, they are already in the DTSI file (same for similar cases in other DTS files).